test/react-15/index.js

```diff
@@ -75,7 +74,6 @@ test.always.afterEach(() => {
     class PureClassComponent extends React.PureComponent {
       render () { throw {error: 'test'} }
     }
-    PureClassComponent.displayName = 'PureClassComponent'
     const result = ReactDOMServer.renderToStaticMarkup(React.createElement(PureClassComponent, {a: 'A'}, 'children'))
     t.is(result, '<div>w00t</div>')
     t.true(guardSpy.called)
```

Comment:
It's worth to add more examples. These lines are responsible for `this.constructor.displayName` test, but there is no tests for `this.constructor.name`